crates/re_log_types/src/data_table.rs

```diff
@@ -624,7 +619,7 @@ impl DataTable {
             timeline: Timeline,
             times: &TimeOptVec,
         ) -> (Field, Box<dyn Array>) {
-            let data = PrimitiveArray::from(times.as_slice()).to(timeline.datatype());
+            let data = DataTable::serialize_primitive_deque_opt(times);
```

Comment:
There! Right there! I done messed up!

```suggestion
            let data = DataTable::serialize_primitive_deque_opt(times).to(timeline.datatype());